js/index.js

```diff
@@ -343,3 +463,9 @@ let runAHP = function(event) {
 
 document.getElementById('calcbtn').addEventListener('click', runAHP);
 document.getElementById('calcbtn').addEventListener('keypress', runAHP);
+
+goalInput.addEventListener('blur', function(event) {
+  saveInputValue(event.target);
+});
+
+setTimeout( function() { clearPairwiseStyleClasses() }, 500 );
```

Comment:
can you add a comment here please on why this is needed? (I'm sure it is, just want to communicate why)

Reply:
Done! The code that requires this step is buried deep in a call chain.